docs/content/contribute/cpo-overrides.md

```
@@ -210,6 +210,38 @@ To run override tests:
+fix from PR #8500 and PR #8512", you can verify the claimed PRs are
+actually present in the image's git history using the
+`/validate-pr-override-images` Claude Code skill.
+
+The PR description must include a structured contract:
+
+```
+branch: 4.20 wants: https://github.com/openshift/hypershift/pull/8593
+branch: 4.21 wants: https://github.com/openshift/hypershift/pull/8593, https://github.com/openshift/hypershift/pull/8565
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)